src/server.tsx

```diff
@@ -245,7 +249,10 @@ app.post(
                 alreadyVisitedCount: payload.targeting.alreadyVisitedCount,
                 countryCode: payload.targeting.countryCode,
                 engagementBannerLastClosedAt: payload.targeting.engagementBannerLastClosedAt,
+                subscriptionsBannerLastClosedAt: payload.targeting.subscriptionsBannerLastClosedAt,
                 isPaidContent: payload.targeting.isPaidContent,
+                remoteSubscriptionsBannerSwitchIsOn:
+                    payload.targeting.remoteSubscriptionsBannerSwitchIsOn,
```

Comment:
This'll need a little update as the key of the switch being sent in the payload from `frontend`/`dcr` is `remoteSubscriptionsBanner` and is being sent in the `switches` property of targeting (https://github.com/guardian/frontend/pull/22805/files#diff-98dfec9cae3fcd83e12f84731e450992R212), so can be accessed as `payload.targeting.switches.remoteSubscriptionsBanner`.

I'd suggest just adding `switches` to the `targeting` Object here eg. `switches: payload.targeting.switches`, so if we add future switches we don't need to keep updating this module.

Reply:
Alternatively, you could just do this:

`remoteSubscriptionsBannerSwitchIsOn: payload.targeting.switches.remoteSubscriptionsBanner`.

Which would involve a little less refactoring!

Reply:
Ah sorry George. I think it's fixed now.